### PR TITLE
Visitor can now choose website language

### PR DESCRIPTION
**Pull request by dominykasmurauskas**

### CODE REVIEW

**Comment by dominykasmurauskas** [comment]
//TODO
Keep language in session attributes after logged-in user logs out.

File: src/EventSubscriber/LocaleSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class LocaleSubscriber implements EventSubscriberInterface
+{
+    private $defaultLocale;
+
+    public function __construct($defaultLocale = 'en')
```
Comment (tadas-nfq) [inline src/EventSubscriber/LocaleSubscriber.php]: Better set default_locale in parameters like email_from. and use it here.

File: templates/base.html.twig [hunk 2]
```diff
@@ -25,6 +25,10 @@
                  </div>
             </div>
         </header>
+        <footer style="position: fixed; bottom: 10%; left: 20px; width: 100%; font-size: 20px">
+            <a href="{{ path('language', {lang: 'lt'}) }}">Lietuviškai</a>
+            <a href="{{ path('language', {lang: 'en'}) }}">English</a>
```
Comment (tadas-nfq) [inline templates/base.html.twig]: Don't forget to move text to translations :)

File: config/packages/security.yaml [hunk 3]
```diff
@@ -20,9 +20,6 @@ security:
             guard:
                 authenticators:
                     - App\Security\ContractorAuthenticator
-            logout:
-                path: app_logout
```
Comment (tadas-nfq) [inline config/packages/security.yaml]: You should add a success_handler key under logout and point it to a service id of a class that implements LogoutSuccessHandlerInterface.

logout:
                path: app_logout
                invalidate_session: false
                success_handler: App\Security\LogoutHandler

File: src/Controller/ContractorAuthController.php [hunk 4]
```diff
@@ -29,9 +30,16 @@ public function login(AuthenticationUtils $authenticationUtils): Response
 
     /**
      * @Route("/logout", name="app_logout")
+     * @param Request $request
+     * @return Response
      */
-    public function logout()
+    public function logout(Request $request): Response
```
Comment (tadas-nfq) [inline src/Controller/ContractorAuthController.php]: Revert changes in logout route

**Pull request merged**